bythepixel/kyrio-go2broadband
# Map Checkbox Values to Bool Values on Form Submit

Closes #15

## How should this be tested?
- Navigate to http://dev.bythepixel.com/kyrio/affiliates
- Fill in some of the form data. Check some (not all) of the checkboxes
- Click 'Next Step'. Copy the url with the form data to an editor
- See all checkboxes present in url data. Checked have true value. Unchecked have false value.

## PR Flow
- [x] Push to Staging
